### PR TITLE
🎨  Refactoring and fix dashboard layout (overflow bug)

### PR DESCRIPTION
**Pull request by Naasgul**
We started this PR by doing some refactoring but ended fixing a layout bug on your dashboard - reviews regarding the refactoring highly appreciated! :) 

PS: The empty` <div> </div>` inside the card-container (dashboard.js) are necessary for fixing the overflow layout bug - currently there isn't a better solution available for it. 

If you run into the same problem, this article may help you: https://alexandergottlieb.com/2018/02/22/overflow-scroll-and-the-right-padding-problem-a-css-only-solution/

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -2,4 +2,5 @@
 build/
 coverage/ 
 storybook-static/
-db.json/
\ No newline at end of file
+./db.json/
+client/db.json
```
Comment (phlgr) [inline .prettierignore:5]: Why do you have two db.json's

File: client/src/pages/Dashboard.js [hunk 2]
```diff
@@ -107,9 +95,11 @@ function Dashboard() {
           subline="Marie Groß, Lucas Naas..."
         />
         <Card bgColor="secondaryDark" imgSrc={Addicon} alt="Plus Icon" />
+        <div></div>
```
Comment (phlgr) [inline client/src/pages/Dashboard.js:98]: Why is this here?

File: client/src/pages/Dashboard.js [hunk 3]
```diff
@@ -107,9 +95,11 @@ function Dashboard() {
...
       </CardContainer>
-      <h3>Mood Survey</h3>
+      <h2>Mood Survey</h2>
       <CardContainer>
+        <div></div>
```
Comment (phlgr) [inline client/src/pages/Dashboard.js:102]:
```suggestion
```

**Review by phlgr — CHANGES_REQUESTED** [review]

File: client/src/pages/Dashboard.js (hunk 2 above)
Comment (Naasgul) [reply client/src/pages/Dashboard.js:98]: This is for fixing the overflow bug left and right of our Card-Container - since we couldn't just add a margin/padding.

File: .prettierignore (hunk 1 above)
Comment (Naasgul) [reply .prettierignore:5]: We weren't sure how to add files to the .prettierignore and tried some cases which we forgot to delete later - thanks for the tip!
